gapic-generator-cloud/templates/cloud/client_test.erb

```diff
@@ -170,48 +78,85 @@ describe Google::Cloud::Speech::V1::SpeechClient do
       }
       uri = "gs://bucket_name/file_name.flac"
       audio = { uri: uri }
+<% end %>
 
       # Create expected grpc response
       expected_response = {}
       expected_response = Google::Gax.to_proto expected_response,
-                                               Google::Cloud::Speech::V1::LongRunningRecognizeResponse
+                                               <%= ruby_namespace namespaces %>::<%= method_name.camelize %>Response
+<% if method_lro? method %>
       result = Google::Protobuf::Any.new
       result.pack expected_response
       operation = Google::Longrunning::Operation.new(
-        name: "operations/long_running_recognize_test",
+        name: "operations/<%= method_name %>_test",
         done: true,
         response: result
       )
+<% end %>
 
       # Mock Grpc layer
+<% if method_lro? method %>
       mock_method = proc do |request|
-        assert_instance_of Google::Cloud::Speech::V1::LongRunningRecognizeRequest, request
-        assert_equal Google::Gax.to_proto(config, Google::Cloud::Speech::V1::RecognitionConfig), request.config
-        assert_equal Google::Gax.to_proto(audio, Google::Cloud::Speech::V1::RecognitionAudio), request.audio
+        assert_instance_of <%= ruby_namespace namespaces %>::<%= method_name.camelize %>Request, request
```

Comment:
We should get this from the proto (can't assume that it follows this naming pattern).